modules/tutorials/examples/logging-aggregator/main.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+echo "Installing ZooKeeper Operator"
+# tag::zk-op[]
+stackablectl release install -i secret -i commons -i zookeeper 23.7
+# end::zk-op[]
+
+# tag::vector-agg[]
+helm install -f vector-aggregator-values.yaml vector-aggregator vector/vector
```

Comment:
This script fails because the Vector aggregator is not ready when the logs are queried.

```suggestion
helm install --wait -f vector-aggregator-values.yaml vector-aggregator vector/vector
```